src/standup/standup.controller.ts

```diff
@@ -30,26 +23,33 @@ export class StandupController {
   @Post()
   @UseGuards(TokenGuard)
   @ApiSecurity('token')
-  createStandup(@Body() body: StandupDTO) {
-    return body;
+  async createStandup(@Body() body: StandupDTO) {
+    return this.standupService.createStandup(body);
   }
 
   @Get()
-  findAllStandups(@User() user) {}
+  async findAllStandups() {
+    return await this.standupService.getAllStandups();
```

Comment:
I wonder if the controller and service methods should be the same when doing the same thing? For example both could be `findAll`, it is in the Standup controller / service already, so not sure that name needs appending

Reply:
updated it

Reply:
Thanks 👍 

Sorry to be a pain, do you think the appended `Standups` is needed? When these are in the class names already, so when we use it is currently `this.standupService.getAllStandups()` but could be shortened to `this.standupService.getAll()` (I prefer `find` tbh but not a biggie)

Reply:
I removed the appended Stanup as mentioned. In the controller function-name I'd leave it in to be honest so its clearer also when not seeing the class name, maybe because its scrolled out.

Reply:
Sure 👍